Makefile

```diff
@@ -36,46 +42,24 @@ lint: node_modules/eslint node_modules/babel-eslint
 
 eslint: lint
 
-example-server:
-	cd examples/server/; $(NPM) install
-	$(NODE) examples/server/index.js
-
-test: build
+test:
 	@$(MOCHA) \
 		--timeout 120s \
 		--slow 3s \
 		--grep "$(FILTER)" \
 		--bail \
 		--reporter spec \
-		build/test
+		--compilers js:babel-register \
```

Comment:
I don't get why we're doing transpilation for the tests again, rather then precompiling them with the makefile?